### PR TITLE
Internal: Adjust drag position based on Auto Flow in Grid Container [ED-10393]

### PR DESCRIPTION
**Pull request by louiswol94**

### CODE REVIEW

**Review by hein-obox — DISMISSED** [review]

File: assets/dev/js/editor/utils/presets-factory.js [hunk 1]
```diff
@@ -305,7 +305,7 @@ presetsFactory = {
 					</g>
 				</svg>
 			`,
-			'3-3': `
+			'2-3': `
```
Comment (louiswol94) [inline assets/dev/js/editor/utils/presets-factory.js:308]: Unrelated fixes in this file.

**Review by udidol — APPROVED** [review]

**Review by davseve — APPROVED** [review]
